app/components/Widget/Voting/Voting.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { BigNumber } from 'bignumber.js'
+import {
+  Card,
+  GU,
+  IconCheck,
+  IconClose,
+  IdentityBadge,
+  ProgressBar,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { useNetwork } from '@aragon/api-react'
+import {
+  VOTING_STATUS,
+  VOTING_STATUS_ONGOING,
+  VOTING_STATUS_REJECTED,
+} from '../../../utils/constants'
+
+const Voting = ({ data }) => {
+  return (
+    <VoteContainer>
+      {data.votes.map(vote => (
+        <Vote key={vote.id}>
+          <UpperBar>
+            <App address={vote.app} />
+            <Status status={vote.status} />
+          </UpperBar>
+          <MiddleBar>
+            <Id id={vote.id}/>
+            <Description text={vote.description} />
+          </MiddleBar>
+          <LowerBar>
+            <Result yea={vote.yea} nay={vote.nay} />
+          </LowerBar>
+        </Vote>
+      ))}
+    </VoteContainer>
+  )
+}
+
+Voting.propTypes = {
+  data: PropTypes.object.isRequired,
+}
+
+const VoteContainer = styled.div`
+  display: flex;
```

Comment:
This style is not enough, the votes cards don't look well and need to wrap.

Look:

![image](https://user-images.githubusercontent.com/5030059/73055916-15c91800-3e8e-11ea-87a6-90ca55abf0a5.png)

Even the overflow is hidden, so the widget becomes unusable

Reply:
Solved partially (now wraps but still needs some care)